mps_youtube/player.py

```diff
@@ -73,6 +73,7 @@ def play(self, songlist, shuffle=False, repeat=False, override=False):
                                                          override=self.override,
                                                          softrepeat=self.softrepeat)
                 self._playsong()
+                self.song_no += 1
```

Comment:
Uh, the changes in this file seem unrelated to the purpose of the commit or PR

Reply:
You should not do this. This is handled by `CmdPlayer.next`

Reply:
You're right, this is unrelated. It should not be here. I'll remove it.